jep/200/README.adoc

```diff
@@ -378,29 +385,60 @@ so the maintainers will be the entry point.
-** The list of tested plugins is provided below
-
-// TODO: Describe what was covered...
-//
-//* Everything in ATH
-//* PCT for a set of plugins, @jglick and @oleg-nenashev will construct a list. Generally, "compatible" and "verified" plugins from https://go.cloudbees.com/plugins/
-//* @oleg-nenashev will review the uncovered plugins, and then we will plan testing for them before the JEP changes get released
-//
-
-=== Plugins verified by PCT
-
-NOTE: The section is in progress
-
-|=========================================================
-| Plugin | Issues found? | Released in | Issue links
-|=========================================================
+* link:https://docs.google.com/document/d/1uQcyaaLvGFwFDe0mQ27JHeG2icdX0XfCHILbHGOtAmA/edit[JEP-200 Testing Notes (BDFL Review)]
+* link:https://github.com/jenkins-infra/jenkins.io/pull/1293[Announcement Blog Post Draft]
+* link:https://wiki.jenkins.io/display/JENKINS/Plugins+affected+by+fix+for+JEP-200[Wiki Page: Plugins affected by JEP-200]
+
+== Appendix A. JEP Review Notes
+
+=== Tested Plugins
+
+During JEP-200 review an extra testing has been performed.
+Testing steps and discovered issues are being tracked in
+link: https://docs.google.com/document/d/1uQcyaaLvGFwFDe0mQ27JHeG2icdX0XfCHILbHGOtAmA/edit[JEP-200 Testing Notes].
+
+* link:https://github.com/jenkinsci/acceptance-test-harness[Jenkins Acceptance Test Harness (ATH)] has been executed with the patched components, several plugins were fixed (see the _Testing_ section)
+** Jenkins WAR from link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] has been tested with a custom core
+** After reviewing of existing ATH tests we concluded that usage of a custom WAR is not a problem
+** We agreed that testing against obsolete dependencies could be a problem.
+During the PR merge procedure the JEP Sponsor and the BDFL Delegate will rerun ATH to confirm there is no issues with stock Jenkins WAR
+* link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)]
+** Originally PCT has been executed for a limited plugin set for a sponsor's custom Jenkins WAR.
+** It was decided that it is not enough (not recent plugin versions, potential impact on the plugin behavior by the custom logic),
+so the BDFL delegate and the JEP Sponsor re-run PCT with a standard Jenkins WAR
+** During testing the BDFL Delegate discovered issues which prevent him from running PCT in
+particular cases.
+These issues are listed in the link:https://issues.jenkins-ci.org/browse/JENKINS-48734[JENKINS-48734] EPIC.
+The blocker issues have been resolved.
+** All plugins recommended in the Jenkins Installation Wizard have been covered as well as many other plugins
+** BDFL delegate has not tested Pipeline and Blue Ocean plugins,
+because they are being maintained by employees of the JEP Sponsor's company.
+According to the JEP sponsor, they were covered by their internal testing procedure.
```

Comment:
There is no “internal testing procedure” here, just

* regular ATH
* regular PCT (including all common Pipeline / Blue Ocean plugins)
* Blue Ocean’s regular acceptance tests

Reply:
OK, still sounds like an internal testing procedure since are results are not public. So I propose to consider it as a blackbox in this JEP